apps/web/src/contexts/theme-context.tsx

```diff
@@ -43,12 +47,16 @@ const ThemeContext = createContext<IThemeContext>({
 export const useThemeContext = () => useContext(ThemeContext)
 
 export const ThemeProvider = ({ children }: { children: React.ReactNode }) => {
-  const [theme, setTheme] = useState<string | null>()
+  const browserTheme = useRef<string | null>(null)
+  const [theme, setTheme] = useState<string | null>(null)
 
   const toggleTheme = useCallback(() => {
     setTheme((theme) => {
-      const actualTheme = theme ?? getThemeFromBrowser()
-      return actualTheme === 'dark' ? 'light' : 'dark'
+      return themeOptions[
+        (themeOptions.indexOf(theme) + 1) % themeOptions.length
```

Comment:
future work: add an alert/toaster context and call:
```toaster.success(`Theme set to ${selectedThemeOption}\`)```

Reply:
It's not, below is just another way to handle CTRL+L.
The current implementation toggles: light -> dark -> OS, whereas the other just toggles between light & dark.
I'll remove the commented out code since I like having the option of getting to the OS option via the keyboard but it won't be obvious which one is the OS option until we get some other visual feedback surfaced to the user. Luckily this is a hidden feature so it doesn't matter much.